app/assets/index.js

```diff
@@ -1,9 +1,11 @@
-import CodeMirror from 'codemirror/lib/codemirror'
```

Comment:
For legibility, I'd keep `import 'codemirror/lib/codemirror'`

Reply:
I don't think so, I guess my ide removed it since it is an unused import. I've added it back